doc/sf/number_series.qbk

```diff
@@ -292,11 +292,12 @@ There are two sets of functions available `prime_sieve` and `prime_range`. `prim
 range 2 to `upper_bound` inclusive. `prime_range` will return all the primes in the range `lower_bound` to `upper_bound`
 inclusive.
 
-If you have a C++17 compatible compiler you are able to pass an execution policy to both both functions. Any
-policy besides `std::execution::seq` will enable internal multi-threading.
+If you have a C++17 compatible compiler you are able to pass an execution policy to both functions. Any
+policy besides `std::execution::seq` will enable internal multi-threading. If your compiler is not C++17 compatible the
+sequential overloads will be used.
```

Comment:
@mborland : I just checked in C++-11 mode with gcc-10, this does not compile. So the correct statement is "This requires C++17".

Reply:
@NAThompson I did not realize that `__has_include()` is a macro introduced in C++17. I am editing the include guards to check for language standard instead. That will allow backwards compatibility to C++11.